defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+urllib3_version: "1.21.*"
+pyOpenSSL_version: "17.0.*"
+ndghttpsclient_version: "0.4.*"
+pyasn1_version: "0.2.*"
+
+python_security_packages:
+    - { name: urllib3, version: "{{ urllib3_version }}" }
+    - { name: pyOpenSSL, version: "{{ pyOpenSSL_version }}" }
+    - { name: ndg-httpsclient, version: "{{ ndghttpsclient_version }}" }
```

Comment:
For our case, it seems as though only `ndg-httpsclient` and `pyasn1` are necessary. If I let the `azavea.docker` example fail by commenting out the installation of this role and only install those two dependencies, the Ansible run completes successfully.